modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserGroupResourceImpl.java

```diff
@@ -244,6 +268,10 @@ public UserGroup putUserGroupByExternalReferenceCode(
 				userGroup.getDescription(), null));
 	}
 
+	private String _formatActionMapKey(String methodName) {
```

Comment:
I would inline this method into a variable. It is called twice, but with the same input parameter.